README.md

```diff
@@ -1,21 +1,39 @@
 # Kopano ONE Container
 
+!! This project is still a work in progress !!
```

Comment:
Seems not really required - i am not a big fan of these markers.

Reply:
Yes, I get your point. I wanted to have a marked to say "this is not yet finished", but on the other hand this could keep people back from providing feedback now that the repo is public.